src/html.js

```diff
@@ -37,8 +37,8 @@ const VOID_ELEMENTS = {}; // poor man's set
 //
 //   without this indirection, `<bar>` would be created before `<foo>`
 export default function generateHTML(tag, params, ...children) {
-	return (stream, nonBlocking, callback) => {
-		stream.write(`<${tag}${generateAttributes(params, tag)}>`);
+	return (stream, { nonBlocking, log = simpleLog }, callback) => {
```

Comment:
to name the export `simpleLog` instead of just `log` ships a personal  meaning without bringing in any value. Instead this reassignment is necessary. IMHO it's is just adding unnecessary noise.